keps/sig-api-machinery/1965-kube-apiserver-identity/kep.yaml

```diff
@@ -19,17 +19,18 @@ see-also:
   - "https://docs.google.com/document/d/1ed7miqlFY7-9lZxE7gzoyx_MFQCtFEDqtcKMpaAmHys/edit?usp=sharing"
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: beta
+stage: stable
 
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.26"
+latest-milestone: "v1.28"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
   alpha: "v1.20"
   beta: "v1.26"
+  stable: "v1.28"
```

Comment:
IMO this is not ready for stable - IIUC nothing actually uses this functionality, so we have no proof or experience that it actually works.  GA graduation criteria should include at least one, ideally two plus usages of this feature (and those usages need to beta level).

The conversation around https://github.com/kubernetes/kubernetes/issues/114314#issuecomment-1352079089 happened so long ago that I have forgotten all the things that we need to fix.

Reply:
> IMO this is not ready for stable - IIUC nothing actually uses this functionality, so we have no proof or experience that it actually works. GA graduation criteria should include at least one, ideally two plus usages of this feature (and those usages need to beta level). 

I think it's reasonable to say this feature shouldn't GA until StorageVersion API is Beta, which is currently the primary use-case for it. I'll add this to the GA criteria.

Reply:
For posterity: UVIP is using this but is still in alpha.